test/select_io_dset.c

```diff
@@ -0,0 +1,3212 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Copyright by The HDF Group.                                               *
+ * All rights reserved.                                                      *
+ *                                                                           *
+ * This file is part of HDF5.  The full HDF5 copyright notice, including     *
+ * terms governing use, modification, and redistribution, is contained in    *
+ * the COPYING file, which can be found at the root of the source code       *
+ * distribution tree, or in https://www.hdfgroup.org/licenses.               *
+ * If you do not have access to either file, you may request a copy from     *
+ * help@hdfgroup.org.                                                        *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+/*
+ * Programmer:
+ *
+ * Purpose:    Tests selection IO for the dataset interface (H5D)
+ */
+
+#include "testhdf5.h"
+#include "H5srcdir.h"
+
+const char *FILENAME[] = {"select_io", /* 0 */
```

Comment:
Is this file created and deleted - or does it need to be added to the build system cleanup sections?
(Also same for parallel test)